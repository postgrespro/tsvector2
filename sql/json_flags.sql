CREATE EXTENSION tsvector2;

select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, '""');
select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, '{}');
select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, '[]');
select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, 'null');
select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, '["all", null]');

DROP EXTENSION tsvector2;
