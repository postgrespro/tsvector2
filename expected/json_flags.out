CREATE EXTENSION tsvector2;
select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, '""');
 jsonb_to_tsvector2 
--------------------
 'aaa':1 'bbb':3
(1 row)

select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, '{}');
 jsonb_to_tsvector2 
--------------------
 'aaa':1 'bbb':3
(1 row)

select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, '[]');
 jsonb_to_tsvector2 
--------------------
 'aaa':1 'bbb':3
(1 row)

select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, 'null');
 jsonb_to_tsvector2 
--------------------
 'aaa':1 'bbb':3
(1 row)

select jsonb_to_tsvector2('english', '{"a": "aaa in bbb", "b": 123, "c": 456, "d": true, "f": false, "g": null}'::jsonb, '["all", null]');
 jsonb_to_tsvector2 
--------------------
 'aaa':1 'bbb':3
(1 row)

DROP EXTENSION tsvector2;
